### DISCUSSION TITLE
[Feature] Update process and scripting

### DISCUSSION THREAD

**Discussion by mprajescu** [mediacms-io/mediacms#29] [Feature Requests]
Continuing the conversation from https://github.com/mediacms-io/mediacms/issues/24

mprajescu commented 7 hours ago
How do you update to a newer version during the current setup? Do a git clone and just run the installer again?

@mgogoulos
 
Contributor
mgogoulos commented 2 hours ago
Hi @mprajescu, we'll update the documentation to include update steps - please move this to the discussions - https://github.com/mediacms-io/mediacms/discussions- and let's continue there

**Comment by phreaknerd** [comment]
Any progress here?

**Comment by mgogoulos** [comment]
There are some notes on https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#2-server-installation and also https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#3-docker-installation

For single server setup through github, just need to get the latest code (eg git pull), then apply migrations (python manage.py migrate) and then restart the services (systemctl restart mediacms celery_short celery_long)
For docker installation, again get the code (eg git pull) , then get new containers (docker pull mediacms/mediacms) and then restart docker-compse (docker-compose down && docker-compose up)

Thinking about it, this doesn't cover the case when MediaCMS is installed through a release :(
